tests/testthat/test-derive_vars_last_dose.R

```diff
@@ -174,3 +174,44 @@ test_that("derive_vars_last_dose when multiple doses on same date - dose_id supp
 
 
 })
+
+
+test_that("An error is issued if same variable is found in both input datasets ", {
```

Comment:
Update `test_that("An error is issued if same variable is found in both input datasets "` to `test_that("derive_vars_last_dose An error is issued if same variable is found in both input datasets "` to be consistent with rest of file.